AppHandling/Run-AlPipeline.ps1

```diff
@@ -528,6 +531,104 @@ function GetInstalledApps {
     Write-GroupEnd
 }
 
+function RunPageScriptingTests {
+    param(
+        [Hashtable] $parameters
+    )
+    $containerName = $parameters.containerName
+    $credential = $parameters.credential
+    $pageScriptingTests = $parameters.pageScriptingTests
+    $restoreDatabases = $parameters.restoreDatabases
+    $pageScriptingTestResultsFile = $parameters.pageScriptingTestResultsFile
+    $pageScriptingTestResultsFolder = $parameters.pageScriptingTestResultsFolder
+    $startAddress = $parameters.startAddress

```

Comment:
Would be cleaner IMO to make these individual parameters rather than part of a parameters hashtable